modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/paths/Testray.path

```diff
@@ -12,12 +12,17 @@
 <tbody>
 <tr>
 	<td>ITEMS_PER_PAGE_BUTTON</td>
-	<td>//div[contains(@class, testray-page)]//div[contains(@class, 'pagination')]//button[contains(@class, 'dropdown')]</td>
+	<td>xpath=(//div[contains(@class, 'pagination')]//button[contains(@class, 'dropdown') and contains(.,'')])[3]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ITEMS_PER_PAGE_BUTTON_IN_PROJECTS</td>
+	<td>xpath=(//div[contains(@class, 'pagination')]//button[contains(@class, 'dropdown') and contains(.,'')])[1]</td>
```

Comment:
Is there a way to avoid using indexes like [1] or [3] here?

Reply:
There is no way to remove the index because the elements are totally equal